.gitmodules

```diff
@@ -103,3 +97,6 @@
 [submodule "vendor/filesystem"]
 	path = vendor/filesystem
 	url = https://github.com/gulrak/filesystem.git
+[submodule "vendor/mapbox-base"]
+	path = vendor/mapbox-base
+	url = https://github.com/mapbox/mapbox-base.git
```

Comment:
Mapbox Base will have to add its submodules as HTTPS in order to make it "bot friendly"

Reply:
https://github.com/mapbox/mapbox-base/pull/1